.github/workflows/tacos_lock.yml

```diff
@@ -70,6 +63,15 @@ jobs:
 
       - id: slices
         run: |
+          sudo sh -c 'ls -d /etc/sudo*' | sudo xargs ls -la
+          sudo sh -c 'find /etc/sudo* -type f' | sudo xargs tail -n9999
+
+          : determine which slices to lock
+          python -c 'import lib; print(lib)' || : code $?
+          echo "$PYTHONPATH"
+          tr ':' '\n' <<< "$PYTHONPATH" |
+            xargs -n1 ls -la
```

Comment:
Which parts of this are debugging and can be cut?

Reply:
deleted; all of that ws meant to be temporary